helmfile/values/prometheus-alerts-wc.yaml.gotmpl

```diff
@@ -1,6 +1,5 @@
 # Note: These values are used for the wc reader. I.e. **in SC** despite the name
 
-esNodeCount: {{ add .Values.elasticsearch.dataNode.count .Values.elasticsearch.clientNode.count .Values.elasticsearch.masterNode.count }}
```

Comment:
Why don't we need this one anymore? To be honest, I'm not really sure why we need them right now but I know there's a reason

Reply:
Because the rule for it is disabled just below it, but I can double check it in case I've missed something.
```yaml
defaultRules:
  rules:
    elasticsearch: false
```

Reply:
Have confirmed, this rule does not get created.